walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -28,44 +42,83 @@
   private static final TimeRange WHOLE_DAY = TimeRange.WHOLE_DAY;
   private static final Comparator<TimeRange> ORDER_BY_START = TimeRange.ORDER_BY_START;
 
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+  public Collection<TimeRange> query(
+      Collection<Event> events, MeetingRequest request) {
+
+    // Array of queries representing times to meet.
+    List<TimeRange> optionalTimeRanges;
 
     // Array of queries representing times to meet.
-    List<TimeRange> query = new ArrayList<>();
+    List<TimeRange> mandatoryTimeRanges;
```

Comment:
Can these be moved lower and joined with their assignments?

Reply:
Joined